src/ApiVersioningExtension/ApiVersioningExtension/Extensions/ApiVersioningDependencyInjectionExtension.cs

```diff
@@ -36,13 +36,13 @@ public static IServiceCollection AddTechBuddyApiVersioning(this IServiceCollecti
         ApiVersioningConfig config = new();
         options.Invoke(config);
 
-        AddApiVersioning(services, config);
+        services.AddApiVersioning(config);
```

Comment:
I used this usage for a purpose. I didn't want to use Extension Methods in an extension method. It would not be more than confusion.